src/main/java/net/atos/zac/documentcreatie/converter/DataConverter.java

```diff
@@ -229,13 +230,22 @@ private String convertToHuisnummer(final Verblijfplaats verblijfplaats) {
     private AanvragerData createAanvragerDataVestiging(final String vestigingsnummer) {
         final ResultaatItem vestiging = kvkClientService.findVestiging(vestigingsnummer);
         if (vestiging != null) {
-            return convertToAanvragerDataVestiging(vestiging);
+            return convertToAanvragerDataBedrijf(vestiging);
         } else {
             return null;
         }
     }
 
-    private AanvragerData convertToAanvragerDataVestiging(final ResultaatItem vestiging) {
+    private AanvragerData createAanvragerDataNietNatuurlijkPersoon(final String rsin) {
+        final ResultaatItem vestiging = kvkClientService.findRechtspersoon(rsin);
+        if (vestiging != null) {
+            return convertToAanvragerDataBedrijf(vestiging);
+        } else {
+            return null;
+        }
+    }
+
+    private AanvragerData convertToAanvragerDataBedrijf(final ResultaatItem vestiging) {
```

Comment:
Is het een optie om de `vestiging != null`-check te doen in deze methode `convertToAanvragerDataBedrijf`? Dat voorkomt een klein beetje duplicate code in het huidige geval en maakt het hergebruiken van de methode gemakkelijker.

Reply:
De controle of vestiging null is net nadat de findXXX methode is aangeroepen vind ik zelf mooier. Gezien bij ons de findXXX null kan teruggeven. Om dit pas een methode verder te doen bij een convert methode vind ik zelf wat minder mooi. 

De if statements kunnen wel omgezet worden naar de short-hand if.
return vestiging != null ? convertToAanvragerDataBedrijf(vestiging) : null;